cmd/cli/server.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2015-2024 MinIO, Inc.
+//
+// This file is part of MinIO Object Storage stack
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"github.com/minio/cli"
+	"github.com/minio/hperf/server"
+	"github.com/minio/hperf/shared"
+)
+
+var (
+	bindFlag = cli.StringFlag{
+		Name:   "bind",
+		EnvVar: "HPERF_BIND",
+		Value:  "0.0.0.0:9010",
+		Usage:  "Hperf will bind to the specified address",
+	}
+
+	storagePathFlag = cli.StringFlag{
+		Name:   "storagePath",
+		EnvVar: "HPERF_STORAGE_PATH",
+		Value:  "$pwd",
```

Comment:
```suggestion
		Value:  "$pwd",
```

You want to show you can show `Getwd()`